### PR TITLE
Resilience message - repositioning and rephrasing the guide message 

### PR DESCRIPTION
**Pull request by Diama1**
#1243

### CODE REVIEW

**Comment by Diama1** [comment]
guidemessage

File: client/src/elm/Pages/Page.elm [hunk 1]
```diff
@@ -204,6 +204,7 @@ type UserPage
     | PatientRecordPage PatientRecordInitiator PersonId
     | MessagingCenterPage
     | WellbeingPage
+    | GuideMessagePage
```
Comment (anvmn) [inline client/src/elm/Pages/Page.elm]: Call the page `MessagingGuide`.

File: client/src/elm/Pages/Router.elm [hunk 2]
```diff
@@ -379,6 +379,9 @@ pageToFragment current =
                 WellbeingPage ->
                     Just "wellbeing"
 
+                GuideMessagePage ->
+                    Just "guide"
```
Comment (anvmn) [inline client/src/elm/Pages/Router.elm]: => `messaging-guide`

File: client/src/elm/Pages/Router.elm [hunk 3]
```diff
@@ -450,6 +453,7 @@ parser =
         , map (\id initiator -> UserPage <| PatientRecordPage initiator id) (s "patient-record" </> parseUuid </> parsePatientRecordInitiator)
         , map (UserPage MessagingCenterPage) (s "messaging-center")
         , map (UserPage WellbeingPage) (s "wellbeing")
+        , map (UserPage GuideMessagePage) (s "guide")
```
Comment (anvmn) [inline client/src/elm/Pages/Router.elm]: => `messaging-guide`

File: client/src/assets/scss/_new.scss [hunk 4]
```diff
@@ -4860,18 +4860,41 @@ div.page-activity.ncd {
...
+      left: 5px;
+      line-height: 30px;
+      position: relative;
+      top: -15px;
+      width: 30px;
+    }
+  }
+}
+
+div.page-activity.guideMessage {
```
Comment (anvmn) [inline client/src/assets/scss/_new.scss]: => `messaging-guide`

File: client/src/elm/Pages/GuideMessage/View.elm [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+                    [ translateText language Translate.GuideMessage
+                    ]
+                , span
+                    [ class "link-back"
+                    , onClick <| SetActivePage <| UserPage WellbeingPage
+                    ]
+                    [ span [ class "icon-back" ] [] ]
+                ]
+    in
+    div [ class "page-activity guideMessage" ]
```
Comment (anvmn) [inline client/src/elm/Pages/GuideMessage/View.elm]: => `messaging-guide`

File: client/src/elm/Pages/GuideMessage/View.elm [hunk 6]
```diff
@@ -0,0 +1,70 @@
+module Pages.GuideMessage.View exposing (..)
```
Comment (anvmn) [inline client/src/elm/Pages/GuideMessage/View.elm]: => `MessagingGuide`

File: client/src/elm/Pages/GuideMessage/View.elm [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+import Html.Attributes exposing (..)
+import Html.Events exposing (onClick)
+import Pages.MessagingCenter.Model exposing (..)
+import Pages.MessagingCenter.Utils exposing (..)
+import Pages.Page exposing (Page(..), UserPage(..))
+import Time
+import Translate exposing (Language, translate, translateText)
+
+
+view : Language -> Time.Posix -> NurseId -> Nurse -> ModelIndexedDb -> Model -> Html Msg
```
Comment (anvmn) [inline client/src/elm/Pages/GuideMessage/View.elm]: You don't use anything besides `Language`.

`view : Language -> Html Msg`

**Review by anvmn — CHANGES_REQUESTED** [review]

File: client/src/elm/App/Fetch.elm [hunk 8]
```diff
@@ -466,6 +466,19 @@ fetch model =
                         )
                     |> Maybe.withDefault []
 
+            UserPage MessagingGuide ->
```
Comment (anvmn) [inline client/src/elm/App/Fetch.elm:469]: As this page presents constant text, there's not need to fetch anything.
So, all this logic should be replaced by `[]`.

File: client/src/elm/App/View.elm [hunk 9]
```diff
@@ -1085,6 +1086,15 @@ viewUserPage page deviceName site features geoInfo reverseGeoInfo model configur
                             |> Html.map (MsgLoggedIn << MsgPageMessagingCenter nurseId)
                             |> flexPageWrapper configured.config model
 
+                    MessagingGuide ->
+                        let
+                            ( nurseId, nurse ) =
+                                loggedInModel.nurse
+                        in
+                        Pages.MessagingGuide.View.view model.language
+                            |> Html.map (MsgLoggedIn << MsgPageMessagingCenter nurseId)
```
Comment (anvmn) [inline client/src/elm/App/View.elm]: Only Msg page needs is `SetActivePage`, which can be used at top level (App.Model).
No need to make use of MessagingCenter page for that. 

See example of Clinical page:

Selection_722
Selection_721

File: client/src/elm/Pages/MessagingGuide/View.elm [hunk 10]
```diff
@@ -0,0 +1,70 @@
+module Pages.MessagingGuide.View exposing (..)
+
+import Backend.Entities exposing (..)
+import Backend.Model exposing (ModelIndexedDb)
+import Backend.Nurse.Model exposing (Nurse)
+import Gizra.NominalDate exposing (fromLocalDateTime)
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import Html.Events exposing (onClick)
+import Pages.MessagingCenter.Model exposing (..)
```
Comment (anvmn) [inline client/src/elm/Pages/MessagingGuide/View.elm]: drop

File: client/src/elm/Pages/MessagingGuide/View.elm [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+import Pages.MessagingCenter.Utils exposing (..)
```
Comment (anvmn) [inline client/src/elm/Pages/MessagingGuide/View.elm]: drop

**Review by anvmn — CHANGES_REQUESTED** [review]

**Review by anvmn — APPROVED** [review]

**Pull request merged**